testData/solutions/win32.json5

```diff
@@ -6278,6 +6278,203 @@
             }
         ]
     },
+    "com.microsoft.windows.colorFilters": {
+        "name": "Windows Built-in Screen Magnifier",
```

Comment:
Well, this is not a magnifier solution, right? :P

Reply:
Whoops! Changing it.